calico-enterprise/network-policy/recommendations/policy-recommendations.mdx

```diff
@@ -60,28 +60,30 @@ The **Policy Recommendations** board is automatically displayed.
 ![Policy-recommendations-board](/img/calico-enterprise/policy-recommendations-board.png)
 
 **Notes**:
+
+- Flow log patterns and processes are continuously monitored for policy recommendations
 - A policy recommendation is generated for every namespace in your cluster (unless namespaces are filtered out by an Admin using the PolicyRecommendationScope resource)
-- Traffic originating from the policy recommendations namespace is used to generate egress rules, and traffic destined for the namespace is used to define ingress rules
-- Flow log patterns and processes are continuously monitored for policy recommendations. As the cluster changes and new namespaces are created, you can get a new policy recommendation. 
+- Recommended policies are continously updated until you **Add to policy board** or **Dismiss policy** using the Actions menu.  
 - Policy recommendations are created as **staged network policies** so you can safely observe the traffic before enforcing them
-- To stop policy recommendations from being processed or updated for a namespace, click the **Action** menu, **Dismiss policy**. 
+- Traffic originating from the policy recommendations namespace is used to generate egress rules, and traffic destined for the namespace is used to define ingress rules  
```

Comment:
I'm wondering if someone might confuse the meaning of the namespace and think that it refers to the tigera-policy-recommendation namespace, which exists to encapsulate all of the policy-recommendation resources

Could it be something like:
` Traffic originating from the recommended policy's namespace is used to generate egress rules, and traffic destined for the namespace is used to define ingress rules`